<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaGraficoConvergencia</class>
 <widget class="QDialog" name="VentanaGraficoConvergencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="layout_grafico"/>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="layout_informacion">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_eje">
         <property name="text">
          <string>Eje de abcisas:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_box_eje">
         <item>
          <property name="text">
           <string>Tiempo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Evaluaciones</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iteraciones</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_mejor_solucion">
       <property name="text">
        <string>Mejor solución: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_peor_solucion">
       <property name="text">
        <string>Peor solución:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
